### PR TITLE
Add new CSV yamls to OLM dirs

### PR DESCRIPTION
**Pull request by ELENAGER**

### CODE REVIEW

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.0/bundle-1.11.0.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+LABEL operators.operatorframework.io.bundle.channel.default.v1=stable
+LABEL operators.operatorframework.io.bundle.channels.v1=stable
+LABEL operators.operatorframework.io.bundle.manifests.v1=manifests/
+LABEL operators.operatorframework.io.bundle.mediatype.v1=registry+v1
+LABEL operators.operatorframework.io.bundle.metadata.v1=metadata/
+LABEL operators.operatorframework.io.bundle.package.v1=ibm-block-csi-operator
+
+COPY manifests /manifests/
+COPY metadata /metadata/
+LABEL com.redhat.openshift.versions="v4.9-v4.11"
```
Comment (matancarmeli7) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.0/bundle-1.11.0.Dockerfile:12]: we will support 4.10-4.12 I think

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.0/metadata/annotations.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+annotations:
+  operators.operatorframework.io.bundle.channel.default.v1: stable
+  operators.operatorframework.io.bundle.channels.v1: stable
+  operators.operatorframework.io.bundle.manifests.v1: manifests/
+  operators.operatorframework.io.bundle.mediatype.v1: registry+v1
+  operators.operatorframework.io.bundle.metadata.v1: metadata/
+  operators.operatorframework.io.bundle.package.v1: ibm-block-csi-operator
+
+  com.redhat.openshift.versions: v4.9-v4.11
```
Comment (matancarmeli7) [inline deploy/olm-catalog/ibm-block-csi-operator/1.11.0/metadata/annotations.yaml:9]: also here 4.10-4.12

**Review by matancarmeli7 — COMMENTED** [review]
You didn't create new CSV files for 1.11 in the [ibm-block-csi-operator-community](https://github.com/IBM/ibm-block-csi-operator/tree/task/CSI-4622_add_new_CSV_yamls_to_OLM_dirs/deploy/olm-catalog/ibm-block-csi-operator-community) directory, can you add 1.11 there also?

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.0/bundle-1.11.0.Dockerfile (hunk 1 above)
Comment (ELENAGER) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.0/bundle-1.11.0.Dockerfile]: Changed to 4.10 - 4.12

File: deploy/olm-catalog/ibm-block-csi-operator/1.11.0/metadata/annotations.yaml (hunk 2 above)
Comment (ELENAGER) [reply deploy/olm-catalog/ibm-block-csi-operator/1.11.0/metadata/annotations.yaml]: Changed to 4.10 - 4.12

File: deploy/olm-catalog/ibm-block-csi-operator-community/1.11.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -0,0 +1,657 @@
+apiVersion: operators.coreos.com/v1alpha1
```
Comment (matancarmeli7) [inline deploy/olm-catalog/ibm-block-csi-operator-community/1.11.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml:1]: could you change the file name to `ibm-block-csi-operator.v1.11.0.clusterserviceversion.yaml` insted of `ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml`?

Comment (ELENAGER) [reply deploy/olm-catalog/ibm-block-csi-operator-community/1.11.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml]: Done

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by matancarmeli7 — APPROVED** [review]
LGTM

**Pull request merged**